components/datadog/apps/logger/images/logger/main.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+)
+
+func main() {
+	handler := http.HandlerFunc(handleRequest)
+	http.Handle("/", handler)
+
+	err := http.ListenAndServe(":3333", nil)
```

Comment:
How do you plan to reach `3333`?

Reply:
I plan to query it from an e2e test in the datadog agent! I've extracted it to a variable